chapters/lexicalstructure.tex

```diff
@@ -247,6 +247,29 @@ \subsection{Strings}\label{strings}
 \end{lstlisting}
 \end{nonnormative}
 
+\subsection{Units of Literal Constants}\label{units-literal-constants}
+
+The following rules apply:
+\begin{itemize}
+\item The result of \lstinline!x + L!, \lstinline!L + x\lstinline!, \lstinline!x - L!, \lstinline!L - x!, \lstinline!x*L!, \lstinline!L*x!, \lstinline!x/L!, where \lstinline!x! is an expression with non-empty \lstinline!unit! attribute string `<s>` and \lstinline!L! is an expression containing only literal constants, shall have the same \lstinline!unit! attribute string of \lstinline!x!.
+\item The result of \lstinline!L/x\lstinline! shall have the \lstinline!unit! attribute string `1/(<s>)`.
+\item The result of \lstinline!x^L! shall have the \lstinline!unit! attribute string of \lstinline!product(fill(x, L))! if L is a positive Integer, or the unit of \lstinline!1/product(fill(x, L))! if L is a negative Integer.
+\item If either side of a relational operator is a literal constant, then it is assumed to have the same \lstinline!unit! attribute string of the other side, if that is well-defined.
```

Comment:
Suggest "If either side of a relational operator is an expression containing only literal constants" to mirror the formulation in rule 1. Otherwise `T > (5+3)` does not match this rule.

Reply:
Unless we find something better than "expression containing only literal constants", that is 😅

Reply:
Of course. But step 1 is to use a consistent formulation in all rules, to then update it in step 2. :-)